contracts/crosschain/axelar/AxelarGatewaySource.sol

```diff
@@ -24,30 +24,31 @@ abstract contract AxelarGatewaySource is IERC7786GatewaySource, AxelarGatewayBas
 
     /// @inheritdoc IERC7786GatewaySource
     function sendMessage(
-        string calldata destination, // CAIP-2 chain ID
-        string calldata receiver, // i.e. address
+        string calldata destinationChain, // CAIP-2 chain identifier
+        string calldata receiver, // CAIP-10 account address (does not include the chain identifier)
         bytes calldata payload,
         bytes[] calldata attributes
     ) external payable returns (bytes32) {
         require(msg.value == 0, "Value not supported");
-        if (attributes.length > 0) revert UnsupportedAttribute(bytes4(attributes[0][0:4]));
+        if (attributes.length > 0)
```

Comment:
We should add a small comment explaining why this error is kept with the `if(...) revert` syntax